index.html

```diff
@@ -0,0 +1,344 @@
+<!DOCTYPE html>
+<html>
+<head>
+		<meta charset="utf-8">
+		<title>Макет</title>
+		<link href="style.css" rel="stylesheet" > 
+		<link rel="stylesheet" href="../Homework/font-awesome-4.7.0/font-awesome-4.7.0/css/font-awesome.min.css">
+</head>
+<header>
+	<div class="header__content">
+		<div class="header__img-wrapper">
+			<img class="header__img" src="./png/AX.png" alt="AX"> 
+			<img class="header__img" src="./png/IT.png" alt="IT">
+   		</div>
+    	<div class="header__menu">
+			<div>Features</div>
+			<div>About</div>
+			<div>Pricing</div>
+			<div>Reviews</div>
+			<div>Contact</div>
+		</div>
+	</div>
+</header>
+<body>
+		<main>
+		<section class="social">
+			<div class="social__content">
+			<div class="social__text-wrapper">
+				<div class="social__title">
+					<p class="social__title">Social Media</p>
+				</div>
+				<div class="social__subtitle">
+					<p class="social__subtitle">Lorem ipsum dolor sit amet, consectetur adipiscing elit. 
+						Aenean euismod bibendum laoreet.</p>
+				</div>
+			</div>
+			<div class="social__icon-wrapper">
+				<div>
+					<i class="fa fa-facebook fa-4x" aria-hidden="true"></i>
+				</div>
+				<div>
+					<i class="fa fa-twitter fa-4x" aria-hidden="true"></i>
+				</div>
+				<div>
+					<i class="fa fa-linkedin-square fa-4x" aria-hidden="true"></i>
+				</div>
+				<div>
+					<i class="fa fa-pinterest-square fa-4x" aria-hidden="true"></i>
+				</div>
+				<div>
+					<i class="fa fa-instagram fa-4x" aria-hidden="true"></i>
+				</div>
+				<div>
+					<i class="fa fa-github fa-4x" aria-hidden="true"></i>
+				</div>
+				<div>
+					<i class="fa fa-rss fa-4x" aria-hidden="true"></i>
+				</div>
+			</div>
+			</div>
+		</section>
+		<section class="tabs">
+			<div>
+				    <div class="tabs__button-wrapper">
```

Comment:
Can you at least understand where your div elements will be ended with such horrible tabulation?